agent/handlers/limit_reached_handler.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/aws/amazon-ecs-agent/agent/logger/audit"
+	"github.com/aws/amazon-ecs-agent/agent/logger/audit/request"
+)
+
+// limitReachedHandler logs the throttled request in the credentials audit log
+func limitReachedHandler(auditLogger audit.AuditLogger) func(http.ResponseWriter, *http.Request) {
```

Comment:
Can this method be added to `agent/handlers/utils/helpers.go`?

Reply:
Yeah it could work in both ways, but I think it falls into the `handler` category, so keep it in the `handlers` package seems more reasonable to me.

Reply:
The `utils` package is also within `handlers`. `limitReachedHandler` seems more of like a helper method. Having a separate file just for it is not of much value imo.